### ISSUE TITLE
NTR: Prenatal ultrasound: oral mass (Lab: ultrasound & echocardiogram)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8698]
New term request Prenatal ultrasound: oral mass (elthat_161002172818):   UMLS: 0577565

**Comment by pnrobinson** [comment]
@MickeySegal
Does this need to be specifically prenatal? What is the context? @sparkste - does this sound like a good prenatal term?

**Comment by MickeySegal** [comment]
We got this from Woodward et al. "Diagnostic Imaging Obstetrics" (3rd edition) on Epignathus. "Teratoma arising from naso- or oropharynx.  fills oral cavity and emanates from mouth &/or nose.... IMAGING: Large, fungating oral mass".

Although this is our only use of the finding, I'd rather not restrict it to teratomas because the clinically relevant finding is the appearance on ultrasound.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000470
- label: Fetal oral mass
- parents: [Abnormal fetal gastrointestinal system morphology (HP:0034207)](https://hpo.jax.org/app/browse/term/HP:0034207)
- pmid: PMID:33904194
- def: A swelling or enlargment localized to the oral cavity of a fetus. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined. This finding is generally first observed by prenatal sonography or magnetic resonance imaging.
- comment: Fetal oral masses are usually observed in the second and third trimesters.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8698
- synonyms:

**Comment by sparkste** [comment]
Hi all,

Apologies for being late to this.

I think it's quite rare that we would use this term. Typically masses around the fetal head and neck are things like teratomas, lymphangiomas, etc, that are usually not in the mouth. Rather, they originate from the face or neck and are often more on one side.

Curious where this term or need for it originated? If there are specific case reports, perhaps we could link to those?

Thanks!
Teresa


Teresa Sparks, MD, MAS
Associate Professor, Maternal-Fetal Medicine and Clinical Genetics
Program Director, Maternal-Fetal Medicine Fellowship
Co-Director, Women's Reproductive Health Research/K12 Program
Division of Maternal-Fetal Medicine and Reproductive Genetics
University of California, San Francisco
________________________________

**Comment by MickeySegal** [comment]
Woodward does not cite references for individual facts, but for 4 "selected references" for the chapter, the following looks relevant:

Allen LM. Prenatal 3-dimensional imaging techniques in the sonographic evaluation of an oral mass: comparison with postnatal imaging modalities. J Ultrasound Med. 2011 Apr;30(4):561-8. doi: 10.7863/jum.2011.30.4.561. PMID: 21460156.

**Comment by sparkste** [comment]
Looks like this is a report of imaging technique for a fetal oral teratoma. There are also rare reports of other types of oral masses ie lingual hamartomas. Perhaps we could make the terms more specific to reflect? I just worry that "oral mass" is non-specific.

Thanks!
Teresa


Teresa Sparks, MD, MAS
Associate Professor, Maternal-Fetal Medicine and Clinical Genetics
Program Director, Maternal-Fetal Medicine Fellowship
Co-Director, Women's Reproductive Health Research/K12 Program
Division of Maternal-Fetal Medicine and Reproductive Genetics
University of California, San Francisco
________________________________
